lib/matplotlib/patches.py

```diff
@@ -1439,7 +1439,7 @@ def __init__(self, xy, width, height, angle=0.0, **kwargs):
         """
         Patch.__init__(self, **kwargs)
 
-        self.center = xy
+        self.xy = xy
```

Comment:
I'd use `self._center` instead of `self.xy`. We shouldn't add a public attribute. While `xy` is used as a variable name in `__init__`, I don't think it's a good choice. "center" is more semantic.

Reply:
Updated to `self._center`, thanks!